general/gecko.cpp

```diff
@@ -0,0 +1,1272 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
```

Comment:
Use the new Copyright header

Reply:
The gecko part of the header also needs updating.  In particular, it currently includes the old LLNL-CODE release number.  See [gecko.h](https://github.com/LLNL/gecko/blob/master/include/gecko.h).

Reply:
I have included the following block in `gecko.{hpp,cpp}` (note the change to "see the above license"):
```
// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and other
// gecko project contributors. See the above license for details.
// SPDX-License-Identifier: BSD-3-Clause
```
However, I don't see any LLNL-CODE number (neither in our `gecko.{hpp,cpp}` nor in `gecko.h`)

Reply:
The (new) LLNL-CODE number is in README.md.  I just wanted the old one taken out.

Reply:
And in case you want to include it, it's LLNL-CODE-800597.

Reply:
Gecko LLNL-CODE is now included in the headers in `gecko.{hpp,cpp}`, thanks.